packages/terra-functional-testing/src/commands/axe/run.js

```diff
@@ -1,40 +1,51 @@
 /* global browser, axe */
 const injectAxe = require('./inject');
 
+/**
+ * Global rule overrides.
+ * Rules modified here will be applied globally for all tests.
+ */
+const GLOBAL_RULE_OVERRIDES = {
+  /**
+   * This rule was introduced in axe-core v3.3 and causes failures in many Terra components.
+   * The solution to address this failure vary by component. It is being disabled until a solution is identified in the future.
+   *
+   * Reference: https://github.com/cerner/terra-framework/issues/991
+   */
+  'scrollable-region-focusable': { enabled: false },
+};
+
 /**
  * Executes axe on the browser.
- * @param {Object} overrides - The axe options.
- * @param {Array} overrides.rules - The rule overrides.
+ * @param {Object} options - The axe options.
+ * @param {Object} options.rules - The rule overrides.
  */
-const runAxe = (overrides = {}) => {
+const runAxe = (options = {}) => {
```

Comment:
Can we bind this to the Terra service so we don't need to pull these config options from a global?

Reply:
I'll experiment with different ways to retrieve the service options in a more direct manner, but I don't think those changes should hold up this PR

Reply:
The viewport helpers and validate elements also need to look up the service options. Making them classes and initialize with the options was not the easiest to do so I set the service options in the [Terra global object](https://github.com/cerner/terra-toolkit/pull/464/files#diff-a6ba3c1d16c08e58e7ee3994f3ab541b63b2a3a8dafb0de9daed2c8090c948d9R41). That's what I going with for now unless we have other better ways to do it.

Reply:
I've created a PR to address these changes, I'd like to move axe configuration discussions onto that PR:

https://github.com/cerner/terra-toolkit/pull/466